docs/content/en/docs/reference/tracing-policy.md

```diff
@@ -73,13 +75,53 @@ metadata:
   name: "fd-install"
 ```
 
-**The specification**
+#### Specification
 
 The `spec` or `specification` can be composed of `kprobes` or `tracepoints` at
 the moment. It describes the arbitrary event in the kernel that will be traced.
+
+##### Kprobes
+
+Kprobes enables you to dynamically break into any kernel routine and collect
+debugging and performance information non-disruptively. kprobes are highly
+tight to your kernel version and might not be portable since the kernel symbols
+depend on your build.
+
+Conveniently, you can list all kernel symbols reading the `/proc/kallsyms`
+file. For example to search for the `write` syscall kernel function, you can
+execute `sudo grep sys_write /proc/kallsyms`, the output should be similar to
+this, minus the architecture specific prefixes.
+
+```
+ffffdeb14ea712e0 T __arm64_sys_writev
+ffffdeb14ea73010 T ksys_write
+ffffdeb14ea73140 T __arm64_sys_write
+ffffdeb14eb5a460 t proc_sys_write
+ffffdeb15092a700 d _eil_addr___arm64_sys_writev
+ffffdeb15092a740 d _eil_addr___arm64_sys_write
+```
+
+You can see that the exact name of the symbol for the write syscall on our
+kernel version is `__arm64_sys_write`. Note that on `x86_64`, the prefix should
+be `__x64_` instead of `__arm64_`.
+
+{{< caution >}}
+Kernel symbols contain an architecture specific prefix when they refer to
+syscall symbols. To write portable tracing policies, i.e. policies that can run
+both on `x86_64` and `arm64`, just use the symbol name without the prefix.
```

Comment:
Won't the same apply to other architectures?

Reply:
Theoretically yes but Tetragon only supports amd64 and arm64 at the moment. It will need some manual changes to support a new arch. Do you have a specific arch in mind?

Reply:
No, I'm just thinking that if sentence doesn't need to be so specific, e.g.

```suggestion
on any architecture, just use the symbol name without the architecture prefix.
```

Reply:
Yes but the reality is that we support only amd64 and arm64 so I'm not sure being implicit about supporting everything would help?

Reply:
I guess it's just a bit more pragmatic, so you don't have to worry about this nuanced documentation goes out of date. But also, as user I wonder if other architectures maybe supported in the future or this policies will only ever work for the two architectures, which is a side effect of  such explicit docs.

Reply:
Ok I changed that to more generic `to run on multiple architectures`

Reply:
Ok I changed that to more generic `to run on multiple architectures`